### DISCUSSION TITLE
Why the nav inconsistency?

### DISCUSSION THREAD

**Discussion by robertandrews** [forem/forem#16959] [Q&A] [answered]
Hi, love your stuff.
I'm really interested in an aspect of your design decision...

**On wide screens...**
On homepage, left-hand nav is the full nav list...
But not on articles or several other pages. On articles, it's only some sharing/bookmarks.

**On mobile widths...**
The hamburger menu reveals the full nav list on all pages.

Why did you decide that the nav options like Home, Reading List, Tags, About etc was only necessary to show on the homepage?

Many sites would figure these to be key nav elements that are required to push people around the site.

And is it counterintuitive that mobile would get the full list on all pages?

Thanks for the chat.

**Comment by ellativity** [comment]
Good question @robertandrews - maybe @ludwiczakpawel knows a little more about the historical context here?

I'm curious to know if your question is coming from a user perspective or a community creator perspective. Are you currently building a community using Forem?

**Reply by robertandrews** [reply]
I'm a Dev.to reader and publisher with an interest in publishing/community platforms.
An admirer of Dev and Forem, watching from afar, potential user.

**Reply by ellativity** [reply]
Thanks, @robertandrews - that's helpful context! Hope you find Paweł's response suitably illuminating.

**Comment by ludwiczakpawel** [comment, marked as answer]
Yes it's all really legacy stuff. This kinds of Information Architecture was there since very early days, before any Product/Design person joined the team. So no one really have thought it through. Now we kinda suffer with what you described. It's definitely not ideal and it needs much more love. Some of the things related to navigation will be addressed fairly soon but changing overall Information Architecture (and really what happens with nav on certain views) is pretty challenging task - not only from Product/Design perspective but also from implementation point of view. Our resources are pretty limited right now so we need to choose carefully what we will be working on next. This "project" would be a very high effort with not such a significant impact. 

But it's definitely a "known issue" - something we think about, something we want to fix eventually but also something quite complex on so many levels.

**Reply by robertandrews** [reply]
Good luck, thanks for the chat.
